code/frameworks/nextjs/src/next-image-loader-stub.ts

```diff
@@ -37,10 +45,16 @@ const nextImageLoaderStub: RawLoaderDefinition<LoaderOptions> = async function N
   let height;
 
   if (extension === 'avif') {
-    const transformer = sharp(content);
-    const result = await transformer.metadata();
-    width = result.width;
-    height = result.height;
+    if (sharp) {
+      const transformer = sharp(content);
+      const result = await transformer.metadata();
+      width = result.width;
+      height = result.height;
+    } else {
+      throw new Error(
+        'You have to install sharp in order to use image optimization features in Next.js.'
+      );
```

Comment:
I wanted to add the part of the documentation for resolving this error like [nextjs messages docs](https://nextjs.org/docs/messages/install-sharp) but I could not find the part to modify in storybook docs. Could you tell me if it exists.

Reply:
You could describe the error in the faq section [here](https://github.com/storybookjs/storybook/blob/next/docs/get-started/nextjs.md#faq)

Reply:
I added the part for resolving the error in the FAQ section you have shown. Please check out: https://github.com/storybookjs/storybook/pull/26787/commits/3eaacff2aeef09fb38fc12d3d886b50d8ee18cab